deepchem/metalearning/maml.py

```diff
@@ -10,60 +10,6 @@
 from deepchem.models.optimizers import Adam, GradientDescent
 
 
-class MetaLearner(object):
-    """Model and data to which the MAML algorithm can be applied.
```

Comment:
This will break the tensorflow code to break because the import will be gone

Reply:
Since there is no circular import Tensorflow tests are running and I have the screenshot of it below.